libutils/ip_address.h

```diff
@@ -92,5 +92,24 @@ bool IPAddressIsIPAddress(Buffer *source, IPAddress **address);
   @return true if a < b, false otherwise.
   */
 bool IPAddressCompareLess(IPAddress *a, IPAddress *b);
+/**
+ * @brief Check if string is a valid representaion of the localhost
+ *        IPv4-/IPv6 addresses.
+ * 
+ *        For IPv4, any address in the range 127.0.0.0/8 is accepted as a
+ *        valid representation of local host. E.g. '127.0.0.1',
```

Comment:
They are not *representations* of localhost, they are just localhost IP addresses. The are separate, different and you can bind/connect to them with different processes and so on.